src/components/vault/EntryDetails.js

```diff
@@ -591,7 +592,7 @@ const FieldRow = ({
       <For each="fieldTypeOption" of={FIELD_TYPE_OPTIONS}>
         <MenuItem
           key={fieldTypeOption.type}
-          text={`Change Type: ${fieldTypeOption.title}`}
+          text={t(`custom-fields.change-type.${fieldTypeOption.title}`)}
```

Comment:
It would be better if this was 2 separate translations, one for "Change Type:" and another for the type itself.